tests/end_to_end/test_tables.py

```diff
@@ -0,0 +1,83 @@
+import pytest  # noqa: E401
+
+import logging
+from pathlib import Path
+import sys
+
+import yaml
+
+from n2y.main import cli_main
+
+
+def test_simple_table(caplog, monkeypatch, request, tmp_path,
+                      valid_access_token):
+    """
+    Simply show what flattened Markdown content is expected if the input
+    contains tables without headers.
+
+    Relies on https://www.notion.so/Simple-Tables-9b1dd705f61647b6a10032ec7671402f?pvs=4
+    """
+    object_id = "9b1dd705f61647b6a10032ec7671402f"
+
+    config = {
+        "exports": [
+            {
+                "id": object_id,
+                "node_type": "page",
+                "output": f"{request.node.name}-output",
+                "pandoc_format": "gfm",
+            }
+        ]
+    }
+    with monkeypatch.context() as m:
+        m.chdir(tmp_path)
+        config_path = Path("{}-config.yaml".format(request.node.name))
+        with config_path.open("w") as fo:
+            yaml.dump(config, fo)
+        m.setattr(sys, "argv", ["n2y", str(config_path)])
+        # cli_main calls sys.exit, which should be changed for testing
+        m.setattr(sys, "exit", lambda x: x)
```

Comment:
We might consider ditching the cli_main wrappers because of the smell of patching sys.exit.